.github/workflows/ci.yml

```diff
@@ -103,9 +103,6 @@ jobs:
         run: (cd packages/dart && dart test)
   coverage:
```

Comment:
I don't think we need a separate CI step for this, coverage should run as part of the test steps. The 2 coverage CI jobs are automatically added by the codecov plugin.

Reply:
Is it possible to upload coverage reports separately?
Thought adding a job is needed due to testing on three platforms.

Reply:
Lets test it? I mean... what could go wrong?

Reply:
I would just move this to the `check-flutter` and `check-dart`.

Reply:
Let's test it, I am still skeptical since check-flutter runs three times (linux, windows, macos)

Reply:
ok, that shouldn't be an issue, covering a line multiple times should be handled fine. total coverage is determined based on PR, not based on workflow step.

Reply:
> Merging [#737](https://codecov.io/gh/parse-community/Parse-SDK-Flutter/pull/737?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=parse-community) (https://github.com/parse-community/Parse-SDK-Flutter/commit/eecb717e2572b9a82abe778acca3fae4bc90d211) into [master](https://codecov.io/gh/parse-community/Parse-SDK-Flutter/commit/1280ce3bf24305aaf24e1d61f4ca792acfcad080?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=parse-community) (https://github.com/parse-community/Parse-SDK-Flutter/commit/1280ce3bf24305aaf24e1d61f4ca792acfcad080) will decrease coverage by 6.14%.

Is my approach just wrong?

Reply:
I think merging the results is necessary 😞 
Therefore a step is needed, that runs after both tests and merges the results created earlier.

Reply:
> The error stack is: Error: No coverage files located, please try use `-f`, or change the project root with `-R`

Never mind